### PR TITLE
fix(url): #MC-219 update front code and API routes

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes

- update front so that routes match with the back-end
- no more platform field

https://confluence.support-ent.fr/display/MAgenda/Calendar+API

## Checklist tests
fill form and submit : there should be no "platform" selection and route should be calendar/url
click on "sync" button : route should be calendar/:id/url

## Issue ticket number and link
[MC-219](https://jira.support-ent.fr/browse/MC-219)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts [hunk 1]
```diff
@@ -2,14 +2,15 @@ import {model, moment, ng} from "entcore";
 import {ROOTS} from "../../core/const/roots";
 import {Calendar} from "../../model";
 import {IScope} from "angular";
+import {Subject} from "rxjs";
 import {ICalendarService} from "../../services";
-import {DateUtils} from "../../utils/date.utils";
+// import {DateUtils} from "../../utils/date.utils";
```
Comment (alicedraillard) [inline src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:7]: for further use ([MC-210](https://jira.support-ent.fr/browse/MC-210))

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts [hunk 2]
```diff
@@ -2,14 +2,15 @@ import {model, moment, ng} from "entcore";
...
 
 interface IViewModel {
     onOpenOrCloseCalendar(calendar: Calendar, savePreferences: boolean): void;
     hideOtherCalendarCheckboxes(calendar: Calendar) : void;
     updateExternalCalendar($event: MouseEvent) : Promise<void>;
-    getLastUpdate(format: string) : string;
+    // getLastUpdate(format: string) : string;
```
Comment (alicedraillard) [inline src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:13]: for further use ([MC-210](https://jira.support-ent.fr/browse/MC-210))

**Review by flmartineau — APPROVED** [review]

**Review by FlorentMr — APPROVED** [review]

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.html [hunk 3]
```diff
@@ -7,12 +7,4 @@
...
            ng-change="vm.hideOtherCalendarCheckboxes(vm.calendar)"/>
     <i class="sync sidebar-title-calendar-display-sync top-spacing-small" ng-click="vm.updateExternalCalendar($event)"></i>
-    <span class="tooltip-top sidebar-title-calendar-display-center-text">
-        <i18n>calendar.last.update</i18n>
-        </br>
-        <div class="tooltip-top-text">
-            <i18n>calendar.recurrence.onlc</i18n> <span>[[vm.getLastUpdate('DD/MM/YYYY')]]</span>
-            <i18n>calendar.search.date.to</i18n> <span>[[vm.getLastUpdate('HH:mm')]]</span>
-        </div>
-    </span>
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/calendar-item/calendar-item.html:17]: Pourquoi est ce que tu supprime cela ?

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts (hunk 1 above)
Comment (BastienBenard) [reply src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:7]: Je comprend pas pourquoi est ce que tu aurais besoin de mettre du code en commentaire pour un future ticket

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import {model, moment, ng} from "entcore";
```
Comment (alicedraillard) [inline src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:1]: not used

**Review by BastienBenard — APPROVED** [review]

**Pull request merged**